scripts/lsync.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
@tengqm if you'd be willing to add it, a brief explanatory comment here would help people who aren't sure what this done.